docs/specification.md

```diff
@@ -112,6 +112,26 @@ We store each chip as geotiff, along with their coordinate & timestamp informati
 
 ![bands](https://github.com/Clay-foundation/model/assets/23487320/85fbc8d2-28f6-4021-855b-c1eb84dd09e3)
 
+### Normalization parameters
+
+To normalize the data before passing it to the model, we computed the following normalization parameters from a random sample of the the training data. The normalization parameters are used in the [Data Module](https://github.com/Clay-foundation/model/blob/main/src/datamodule.py#L108), for partial
```

Comment:
Best to link to a tag reference (v0.0.1) instead of the `main` branch, because the line number might change over time.

```suggestion
To normalize the data before passing it to the model, we computed the following normalization parameters from a random sample of the training data. The normalization parameters are used in the [Data Module](https://github.com/Clay-foundation/model/blob/v0.0.1/src/datamodule.py#L108), for partial
```